shared/js/background/atb.es6.js

```diff
@@ -126,6 +126,7 @@ const ATB = (() => {
         // builds a new query string containing only accepted params
         getAcceptedParamsFromURL: (url) => {
             const validParams = new URLSearchParams()
+            if (url === '') return validParams
```

Comment:
Not sure I understand what could cause url to be blank here, since it's pulling from [getDDGTabUrls](https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/develop/shared/js/background/chrome-wrapper.es6.js#L41). @Charlie-belmer did you encounter this case while testing?